api/spec/v1.yaml

```diff
@@ -1063,7 +1124,7 @@ components:
 
     ConsensusEvent:
       type: object
-      required: [height, type, body]
+      required: [block, type, body]
```

Comment:
this is unrelated to the PR; will move it to a PR on mitja's branch for fixing api spec

Reply:
Thanks! It was already fixed in #280 though, I thought.